#ifndef LC_MERGE_H_
#define LC_MERGE_H_

#include <math.h>
#include <iostream>
#include <fstream>

#include <ros/ros.h>
#include <camera_calibration_parsers/parse_ini.h>

#include <pcl/point_cloud.h>
#include <pcl/point_types.h>
#include <pcl_ros/point_cloud.h>
#include <pcl/common/transforms.h>

#include <image_transport/image_transport.h>
#include <cv_bridge/cv_bridge.h>
#include <sensor_msgs/image_encodings.h>
#include <sensor_msgs/PointCloud2.h>
#include <opencv2/highgui/highgui.hpp>
#include <opencv2/calib3d/calib3d.hpp>

#endif
